.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:12.13.1-browsers
+      - image: cimg/node:14.19-browsers
```

Comment:
Thanks for updating to `cimg`. But node 14 is still pretty old, most of our projects have moved on to at least 16, some to 18 (and now I see 20 is out even).

Reply:
I totally agree that 14 is old. IIRC this was an easy upgrade and upgrading to anything newer turned out to be non-trivial. I figured it doesn't matter that much since it's only used for CI. I can sure take a look at it though.

Reply:
Old Storybook doesn't want to work with newer Node. I've started the upgrade process. It's indeed non-trivial.

Reply:
ah ok - would be nice to update for sure but if this is too onerous I won't insist.

Reply:
Yeah, I'd argue it's best to delay this upgrade until https://github.com/storybookjs/storybook/issues/9828 is resolved. That's supposed to happen with Storybook V8 release.

As of now I've sunk a couple of hours into getting Storybook V7 to work. The result is pretty ugly (you can see it in my last commit). And now that Storybook works, Percy refuses to work, so I opened a support ticket with Percy to sort it out.

Node version in CI doesn't affect us much, as I haven't yet bumped into any packages that dropped support for it (except for Storybook).

I can take that Storybook commit and cherry-pick it into a separate PR in order not to loose that work.

Reply:
Percy fixed by downgrading it. So we have a build. I'll let you decide if you wanna keep this Node + Storybook upgrade.

Reply:
Thanks for fighting through that - let's keep to the latest Node + Storybook, seems like that's the bigger change than having to push Percy back a minor and will keep us closer to the latest versions overall.